packages/e2e-playwright/models/card.ts

```diff
@@ -96,13 +118,37 @@ class Card extends Base {
         this.cvcInput = cvcIframe.locator(`input[aria-label="${CVC_IFRAME_LABEL}"]`);
         this.cvcIframeContextualElement = cvcIframe.locator('.aria-context');
 
+        /**
+         * HolderName elements, in Checkout
+         */
+        this.holderNameField = this.rootElement.locator('.adyen-checkout__card__holderName'); // Holder
+        this.holderNameLabelWithFocus = this.holderNameField.locator('.adyen-checkout__label--focused');
+
         /**
          * Installments related elements
          */
         this.installmentsPaymentLabel = this.rootElement.getByText(INSTALLMENTS_PAYMENTS);
         this.revolvingPaymentLabel = this.rootElement.getByText(REVOLVING_PAYMENT);
         this.installmentsDropdown = this.rootElement.locator('.adyen-checkout__dropdown__button');
         this.selectorList = this.rootElement.getByRole('listbox');
+
+        /**
+         * Social Security Number related elements
+         */
+        this.ssnField = this.rootElement.locator('.adyen-checkout__field--socialSecurityNumber'); // Holder
+        this.ssnLabelWithFocus = this.ssnField.locator('.adyen-checkout__label--focused');
+
+        /**
+         * AVS related elements
+         */
+        this.addressField = this.rootElement.locator('.adyen-checkout__field--street'); // Holder
```

Comment:
We already have the model `card-avs.ts`, which contains the address field on it.

My question here is if we are going to add everything to this model (avs, kcp, address lookup, etc..) or if we are going to split in different models which each will contain their own custom logic/locators. 

I don't have a strong preference tbh, but we need to decide. We already started splitting things (as we have with `card-avs`) but we can always go back if we think that it is not needed.

Reply:
I believe the `CardWithAvs` class has this selector - `await cardAvsPage.billingAddress.streetInput`?
Regarding the `focus`, wouldn't this code work? 
```ts
await expect(cardAvsPage.billingAddress.streetInput).toBeFocused();
```

Reply:
@ribeiroguilherme I'll make use of the other models / split into other models